keel-bom/keel-bom.gradle.kts

```diff
@@ -0,0 +1,49 @@
+plugins {
+  `java-platform`
+  `maven-publish`
+}
+
+// without this building the pom fails when using the Nebula publishing plugin
+configurations {
+  create("compileOnly")
+}
+
+javaPlatform {
+  allowDependencies()
+}
+
+if (property("enablePublishing") == "true") {
+  publishing {
+    publications {
+      named<MavenPublication>("nebula") {
+        from(components["javaPlatform"])
+      }
+    }
+  }
+}
+
+dependencies {
+  api(platform("com.netflix.spinnaker.kork:kork-bom:${property("korkVersion")}"))
+  constraints {
+    api(project(":keel-api"))
```

Comment:
I feel like you could possibly programmatically generate this list by getting the rootProject and enumerating the subprojects excluding this project - might be worth a look so that forgetting to add a new module to this list doesn't become a thing?